BaSD-2021/German-MindSet-app
# BaSD-MS-34: Clients redux

## Description:
Implementation of Redux to the project made with React.
The Create, Update and Delete functions of the Clients resource were adapted to Redux.
### Steps to test Client's functions:
1. Run `npm run start` on both front and back-end apps.
2. Search for **Clients** option on nav bar and click on it.
3. Click **add** or **edit** button to open form.
4. Complete all the fields with the corresponding data. 
If you are trying to update a client, its information will be loaded in the form automatically. Just change the necessary data.
If you are trying to add a new client, just complete with all the required information.
5. Click "Confirm" button.
6. Check if new client is shown in the Client's list or in case of having edited a client, check if the changed information is correct.
### Reviewers:
- @germanradium 
- @nicolasblasco 
- @GuidoGlielmi 
- @ValentinaCollareta
